build/__variables.sh

```diff
@@ -40,9 +40,15 @@ declare -r ACR_RUNTIME_IMAGES_REPO="$ACR_PUBLIC_PREFIX"
 declare -r ACR_PACK_BUILDER_IMAGE_REPO="$ACR_PUBLIC_PREFIX/$PACK_BUILDER_IMAGE_NAME"
 declare -r ACR_PACK_IMAGE_REPO="$ACR_PUBLIC_PREFIX/$PACK_IMAGE_NAME"
 declare -r ACR_PACK_STACK_BASE_IMAGE_REPO="$ACR_PUBLIC_PREFIX/$PACK_STACK_BASE_IMAGE_NAME"
+declare -r AGENT_BUILD=""
 
 # Flag to add information to images through labels (example: build number, commit sha)
 declare -r EMBED_BUILDCONTEXT_IN_IMAGES="${EMBEDBUILDCONTEXTINIMAGES:-false}"
 declare -r GIT_COMMIT=$(git rev-parse HEAD)
 
 declare -r DOCKER_SYSTEM_PRUNE="${ORYX_DOCKER_SYSTEM_PRUNE:-false}"
+
+# If build_number has value that means we are building in build agent and not locally
+if [ -n "$BUILD_NUMBER" ]; then
+    AGENT_BUILD="true"
```

Comment:
`declare -r AGENT_BUILD="true"`

Reply:
@kichalla I declared it as "" by default and only for build's in build-agent it will be true. 
`declare -r AGENT_BUILD="" `

Reply:
still, it would probably be better if it was read-only in this case as well